src/QMCWaveFunctions/tests/CMakeLists.txt

```diff
@@ -51,8 +52,8 @@ IF (NOT QMC_CUDA)
   ENDIF()
 ENDIF()
 
-SET(TRIALWF_SRC test_wf.cpp test_TrialWaveFunction.cpp test_wavefunction_factory.cpp
-                test_TrialWaveFunction_diamondC_2x1x1.cpp test_TrialWaveFunction_He.cpp
+SET(TRIALWF_SRC test_wf.cpp FakeSPO.cpp test_TrialWaveFunction.cpp test_wavefunction_factory.cpp
```

Comment:
probably will just merge test_TrialWaveFunction.cpp and test_TrialWaveFunction_diamondC_1x1x1.cpp back together since it turned out to be basically impossible to write decent unit tests without the xml and hdf5 mess.